### DISCUSSION TITLE
Is it necessary to include TXA segment in MDM message.

### DISCUSSION THREAD

**Discussion by BforAzure2** [nextgenhealthcare/connect#4812] [Q&A] [answered]
Hi Mirth Gurus, 

We have an outbound MDM T02. But we are sending doc base64 in OBX and feel there is no need of TXA. Is it a violation of HL7 MDM if we exclude TXA from MDM or we should include it by any means?

**Comment by pacmano1** [comment]
For the most part any quesiton like yours is really subject to the party with whom you are integrating.

**Reply by BforAzure2** [reply]
@pacmano1 the party is flexible and happy to mold the stuff as per our template. We just want to make sure that we are not violating any HL7 rules when deciding a format for our party.

**Reply by pacmano1** [reply]
The formal spec says it is required for example in v2.5.  

See https://hl7-definition.caristix.com/v2/HL7v2.5/TriggerEvents/MDM_T02

**Reply by pacmano1** [reply]
And given it takes literally 30 seconds to add, we both spent more time on this question than you adding the code, lol.

**Reply by BforAzure2** [reply]
Thanks @pacmano1 - No the purpose of the question was why we should add this when our purpose is fulfilled with OBX segment. Yes, we read that it is required in 2.3 and 2.5 - but what is the catch if for example we do not include. 

Adding it is not difficult but we find it unnecessary that's why confuse.

**Reply by pacmano1** [reply]
Like I said for the most part any quesiton like yours is really subject to the party with whom you are integrating.   You omit it this time and next time some other partner wants it.

**Reply by BforAzure2** [reply]
Got it- Thanks @pacmano1

**Reply by tonygermano** [reply]
Some parsers are more forgiving than others when handling messages that do not conform to the standard.

**Comment by pacmano1** [comment, marked as answer]
Answered in thread
